templates/zerver/plans.html

```diff
@@ -81,18 +81,22 @@ <h2>Standard</h2>
                                             $8/month billed monthly
                                         </div>
                                     </div>
-                                    {% if realm_plan_type in [3, 4] %}
-                                    <a href='/upgrade' class="button black-current-value" type="button">
-                                        Current plan
-                                    </a>
-                                    {% elif realm_plan_type == 0 %}
+                                    {% if not realm %}
                                     <a href="/new" class="button green">
                                         {% if free_trial_days %}
                                         Start {{ free_trial_days }} day free trial
                                         {% else %}
                                         Buy Standard
                                         {% endif %}
                                     </a>
+                                    {% elif realm.plan_type in [realm.STANDARD, realm.STANDARD_FREE] %}
+                                    <a href='/billing' class="button black-current-value" type="button">
+                                        {% if realm_on_free_trial %}
+                                        Current plan (free trial)
+                                        {% else %}
+                                        Current plan
+                                        {% endif %}
+                                    </a>
```

Comment:
Can we merge this (displaying free trial status) as a prep commit?  Would be nice to get clear improvements like this in rather than having it stacked on a giant commit that may take a few rounds.